diff --git a/lib/parse.js b/lib/parse.js
index bd3b8ccb6feff4e430c2f248134b4d6fe05bccf7..90fad211679263bd942f452300a2d920e2d7a774 100644
--- a/lib/parse.js
+++ b/lib/parse.js
@@ -25,9 +25,6 @@ function Parse(opts) {
   self._opts = opts || { verbose: false };
 
   PullStream.call(self, self._opts);
-  self.on('finish',function() {
-    self.emit('close');
-  });
   self._readRecord().catch(function(e) {
     if (!self.__emittedError || self.__emittedError !== e)
       self.emit('error',e);
